tests/apollo/test_skvbc_long_running.py

```diff
@@ -31,13 +34,17 @@ def start_replica_cmd(builddir, replica_id):
     """
     statusTimerMilli = "500"
     viewChangeTimeoutMilli = "10000"
+    autoPrimaryRotationTimeoutMilli = "10000"
     path = os.path.join(builddir, "tests", "simpleKVBC", "TesterReplica", "skvbc_replica")
     return [path,
             "-k", KEY_FILE_PREFIX,
             "-i", str(replica_id),
             "-s", statusTimerMilli,
             "-v", viewChangeTimeoutMilli,
-            "-p"]
+            "-a", autoPrimaryRotationTimeoutMilli,
```

Comment:
no, please, not the "-a" :) it's not used anywhere for now - it's a kind of possible, but unsupported feature

Reply:
let's do view change in a separate PR

Reply:
yeah, you right I forgot this from other tests I've made.